tron/mesos.py

```diff
@@ -226,7 +240,16 @@ def get_runner(self, mesos_address, queue):
                 'framework_name': 'tron',
             }
         )
-        return Subscription(executor, queue)
+        logging_executor = self.processor.executor_from_config(
+            provider='logging',
+            provider_config={
+                'downstream_executor': executor,
+                'destination': 'python',
```

Comment:
as mentioned in related task_processing PR - i'd prefer to pass a logger in here directly:

    'destination': lambda tid, msg, str: logging.getattr('error' if stream == 'stderr' else 'info')(msg)

or something